text/src/autogluon/text/automm/presets.py

```diff
@@ -0,0 +1,66 @@
+import os
+from .constants import (
+    MODEL,
+    DATA,
+    OPTIMIZATION,
+    ENVIRONMENT,
+)
+
+
+def list_model_presets():
+    """
+    List all available model types.
+    Image/text backbones can be customized for one model type.
+
+    Returns
+    -------
+    A list of model types.
+    """
+    cur_path = os.path.dirname(os.path.abspath(__file__))
+    model_config_dir = os.path.join(cur_path, "configs/model")
```

Comment:
Change to `os.path.join(cur_path, "configs", "model")` Otherwise, this won't work with windows

Reply:
Done.